src/components/NotificationsContext/NotificationsContext.jsx

```
@@ -0,0 +1,91 @@
+              typeof props.label === 'string' ? props.label : undefined
+            }
+            autoCloseDuration={AUTO_CLOSE_DURATION}
+            autoClose={true}
+            key={props.id}
+            {...props}
+            size={NotificationSize.Medium}
+            style={getNotificationStyle(heights, index)}
+            position="top-right"
+            onClose={() =>
+              setNotifications((items) =>
+                items.filter((i) => i.id !== props.id)
```

ESLint

[error] Refactor this code to not nest functions more than 4 levels deep.

(sonarjs/no-nested-functions)